--- a/source/common/grpc/google_async_client_impl.h
+++ b/source/common/grpc/google_async_client_impl.h
@@ -274,7 +274,7 @@
     PendingMessage(Buffer::InstancePtr request, bool end_stream);
     // End-of-stream with no additional message.
     PendingMessage() = default;

-    const absl::optional<grpc::ByteBuffer> buf_;
+    const absl::optional<grpc::ByteBuffer> buf_{};
     const bool end_stream_{true};
   };
